paper/paper.md

```diff
@@ -27,6 +27,25 @@ bibliography: paper.bib
 * introduction to package
 * maybe quick example
 
+section 1:
+Intro, statement of need
+
+HELP NEEDED: idea collection for statement of need
+* Paragraph1: focus on need for spatial analysis?
+
+* Paragraph2: focus on need to visualize spatial analysis?
+
+* paragraph3: splot as solution, this is what we can do for different audiences
+
+splot is a visual steering tool, facilitating human guidance in interpreting results and choosing between different statistical analysis methods.
+
+* paragraph4: more technical description
+
+Implemented in Python, splot extends both PySAL's analytics and matplotlib visualization functionality in visualising spatial analysis workflows and result presentation. It provides users quick access through a high level API to visualise popular PySAL objects. We hereby aim to implement a multilute of views for different spatial analysis workflows, to provide users the opportunity to view a problem from different angles. The splot package facilitates the creation of both static plots ready for publication and interactive visualisations for quick iteration and spatial data exploration. 
```

Comment:
...implement a **multitude** of views...

Reply:
thank you!